app/controllers/users/verify_account_controller.rb

```diff
@@ -42,5 +43,10 @@ def decrypted_pii
         cacher.fetch
       end
     end
+
+    def next_step
+      return after_sign_in_path_for(current_user) if user_session['openid_auth_request'].present?
+      sign_up_completed_url
```

Comment:
I don't understand why OpenID Connect users would not be presented with the completions page. Could you explain that to me please?